### ISSUE TITLE
User Are Not Showing

### ISSUE THREAD

**Issue by Rk5050** [EVLF/Cypher-Rat-Source-Code#79]
User are not showing but only listed in logs with ip but not showing device in user menu. How fix it ?

**Comment by the-at** [comment]
do you give accessibility permission? and what model device are you trying?

**Comment by evenkat295** [comment]
yes connection is coming but users are not showing just blank on users area please solve it

**Comment by evenkat295** [comment]
> User are not showing but only listed in logs with ip but not showing device in user menu. How fix it ?

if you have fix this issue share the solution with me

**Comment by kretcz** [comment]
and with me too
